onnxruntime/test/python/transformers/test_generation.py

```diff
@@ -394,5 +394,95 @@ def test_cross_qk_overall(self):
         self.run_configs(decoder_input_ids)
 
 
+class TestBeamSearchWhisperOpenai(unittest.TestCase):
```

Comment:
This looks identical to `TestBeamSearchWhisper` except for the arguments passed in the test case. Let's add this as another test case here (ex: `test_openai_impl`) and remove this class.

https://github.com/microsoft/onnxruntime/blob/d611b2e81933294e1175ce5702a79e458781516d/onnxruntime/test/python/transformers/test_generation.py#L366-L394